### ISSUE TITLE
Training of the cascading DDPM

### ISSUE THREAD

**Issue by royzhang12** [lucidrains/imagen-pytorch#66]
Hi, 

Thank you for your excellent implementation.

Just a small but vital question for the training detail. To train the cascaded Imagen model, do I need to try the low resolution model until it converges, then use that model to train the high resolution model. Or should I train them simuteneoly  for each mini-batch?

To be more clear for my quesition, for the first way of training, I mean first train the text to 64x64 low resolution model on the whole dataset for enought epoches until it converges. Then based on the well trained model to train the high resoltuoin model.

For the simutaneously trainng way, i mean for each minibatch, first backward the loss from the low resolution model and then backwards the loss for the high resoluton model and then repeated the steps for all the minibach of a datasets.

**Comment by lucidrains** [comment]
@royzhang12 the great thing about cascading DDPM is that you can train each unet separately, completely out of order, and then chain them together later
